utils/xmtpRN/conversations.ts

```diff
@@ -787,7 +787,9 @@ export const sortRequestsBySpamScore = (
   requests.forEach((conversation) => {
     const isLikelyNotSpam =
       conversation.spamScore !== undefined &&
-      (conversation.spamScore === null || conversation.spamScore < 1);
+      (conversation.spamScore === null || conversation.spamScore < 1) &&
+      conversation.version !== ConversationVersion.GROUP;
+    // @todo => remove this once we have a spam score for groups
```

Comment:
you have effectively implemented a spam score for groups here right? just based on the groupAddedBy spam score?

Reply:
true! will clarify the comment: we need to think through whether the same scoring logic for DMs should also apply to groupAddedBy senders